src/fu-util.c

```diff
@@ -2204,6 +2204,34 @@ fu_util_downgrade (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_update_device_with_release (priv, dev, rel, error);
 }
 
+static gboolean
+fu_util_activate (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GPtrArray) devices = NULL;
+
+	/* check args */
+	if (g_strv_length (values) > 0) {
```

Comment:
Maybe we want to allow the user to do `fwupdmgr activate deadbeef` to just activate one device, and default to no args meaning "all"?

Reply:
Yeah it acts more like 'install' does then too.